Reduce set of org admins

accorvin opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

dchourasia commented:
@accorvin wondering if we can remove the bots as well from the admin list which is consistent with ODH org

accorvin commented:
@dchourasia I've moved the two bots from admins and moved to just members as you suggested. I've additionally added the bots to the rhods-operator-maintainers team as that rhods-operator repo appears to be the only place where we need elevated permissions for these bots. I believe this change is good to merge now.

dchourasia commented:
@accorvin thanks
